README.md

```diff
@@ -131,6 +131,10 @@ To start proxy and other required services
 ```
 [user@localhost ~]$ export DSAAS_PREVIEW_TOKEN=<dsaas-preview-token> 
 [user@localhost ~]$ ./scripts/setupLocalProxy.sh start 
+```
+Run command below this in a seperate terminal so that we can see logs of the above command. 
+```
+[user@localhost ~]$ export DSAAS_PREVIEW_TOKEN=<dsaas-preview-token> 
```

Comment:
we should probably tell people to put that in their `~/.bash_profile` instead and not to export it everytime you are starting a proxy command,